src/apps/ociodisplay/main.cpp

```diff
@@ -129,13 +129,20 @@ static void InitImageTexture(const char * filename)
             img.resize(texWidth*texHeight*components);
             memset(&img[0], 0, texWidth*texHeight*components*sizeof(float));
 
-            f->read_image(
-#if (OIIO_VERSION >= 10800)
+            const bool ok = f->read_image(
+#if OIIO_VERSION >= 10800
                 OIIO::TypeFloat, 
 #else
                 OIIO::TypeDesc::TypeFloat, 
 #endif
                 &img[0]);
+
+                if(!ok)
```

Comment:
Lines 140-144 indentation should match line 132

Reply:
Fixed